packages/lodestar/src/sync/reqResp/reqResp.ts

```diff
@@ -97,11 +98,11 @@ export class BeaconReqRespHandler implements IReqRespHandler {
       case Method.Ping:
         return await this.onPing(request as ReqRespRequest<Ping>, peerId, sink);
       case Method.Metadata:
-        return await this.onMetadata(request as ReqRespRequest<null>, peerId, sink);
+        return await this.onMetadata(request as ReqRespRequest<null>, sink);
       case Method.BeaconBlocksByRange:
-        return await this.onBeaconBlocksByRange(request as ReqRespRequest<BeaconBlocksByRangeRequest>, peerId, sink);
+        return await this.onBeaconBlocksByRange(request as ReqRespRequest<BeaconBlocksByRangeRequest>, sink);
       case Method.BeaconBlocksByRoot:
-        return await this.onBeaconBlocksByRoot(request as ReqRespRequest<BeaconBlocksByRootRequest>, peerId, sink);
```

Comment:
I would keep peerId on those methods. We could/should track those requests by peerId so we can blacklist peer if requests too much blocks or something